osf/migrations/0241_merge_20211116_2115.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2021-11-16 21:15
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0239_abstractprovider_allow_bulk_uploads'),
```

Comment:
Can we remake the migration instead of making a merge migration?

Reply:
> It would be nice if we can avoid adding unnecessary merge migrations to the codebase.

Good point! I chatted with @mfraezz on this one a while ago. At that time, we decided not to do a staging2 rollback and thus went with the merge commit.

Reply:
> Can we remake the migration instead of making a merge migration?
Let's re-do the migration and roll back staging2 before merging to `develop`.